charts/wharf-helm/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: wharf-helm
 description: Deploy Wharf to Kubernetes
 type: application
-version: 3.0.0
+version: 3.0.1
```

Comment:
IMO this is a minor version bump, as you've added functionality.

Suggest bumping to v3.1.0 instead